### ISSUE TITLE
Task 2b - MLP

### ISSUE THREAD

**Issue by Zoyela** [Zoyela/PR19-tandomeijivan#2]
• Apply MLP to the MNIST dataset
• MLP with one hidden layer
• Optimize parameters using cross-validation
• Output
• Plot showing error-rate on training and validation set
with respect to the training epochs
• Accuracy on the test set with optimized parameters
